<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="teSourceCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="MyGLWidget" name="wGLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teErrorsAndWarnings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>500</width>
      <height>200</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teRimal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>500</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblErrorsAndWarnings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Errors &amp; warnings:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRimal">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>520</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rimal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTakenCoins">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Taken Coins: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompile"/>
   <addaction name="actionReloadEnvironment"/>
  </widget>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/images/green arrow right.png</normaloff>:/images/green arrow right.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionReloadEnvironment">
   <property name="icon">
    <iconset resource="MyRes.qrc">
     <normaloff>:/images/reload arrow.png</normaloff>:/images/reload arrow.png</iconset>
   </property>
   <property name="text">
    <string>Reload Environment</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyRes.qrc"/>
 </resources>
 <connections/>
</ui>
